utils/json-utils/src/main/scala/pl/touk/nussknacker/engine/json/swagger/SwaggerTyped.scala

```diff
@@ -50,6 +52,7 @@ object SwaggerTyped {
       case Some(ref) =>
         SwaggerTyped(swaggerRefSchemas(ref), swaggerRefSchemas)
       case None => (extractType(schema), Option(schema.getFormat)) match {
+        case (None, _) if !schema.getAnyOf.isEmpty => swaggerUnion(schema, swaggerRefSchemas)
```

Comment:
What about such case:
https://json-schema.org/understanding-json-schema/reference/combining.html#factoring-schemas? how will it be handled?

Reply:
added tests in [SwaggerBasedJsonSchemaTypeDefinitionExtractorTest.scala](https://github.com/TouK/nussknacker/pull/3687/files#diff-4d486ce7f1bd5033ddf7cf294c280cdf3ad63fca46611668e6526ada9595368c) whic cover this case

Reply:
:+1: